apps/desktop/src/test/effect-test-utils.ts

```
@@ -0,0 +1,334 @@
+    })
+  )
+  
+  await Effect.runPromise(program)
+  
+  // Verify cleanup was called
+  expect(cleanupCalled).toBe(true)
+  await cleanupCheck()
+}
+
+// STM test helper
+export const runSTM = <A, E = never>(
```

Biome (2.1.2)

[error] Shouldn't redeclare 'runSTM'. Consider to delete it or rename it.

'runSTM' is defined here:


(lint/suspicious/noRedeclare)